### PR TITLE
[Feature for #118] Creates group->remove, to remove repo(s) from a group

### PR DESCRIPTION
**Pull request by tmack0**
Create group subcommand 'remove', to remove a repo or list of repos from an existing group.
If group or repo doesn't exist, does nothing.
Does not remote the repo from gita repos file.

https://github.com/nosarthur/gita/issues/118

### CODE REVIEW

**Comment by tmack0** [comment]
will see if I can craft a test to improve coverage. 1st attempt shows that current tests will try to use local gita envs (was failing with a group name I have in use).

File: gita/__main__.py [hunk 1]
```diff
@@ -363,6 +371,17 @@ def main(argv=None):
                     metavar='group-name',
                     required=True,
                     help="group name")
+    pg_remove = group_cmds.add_parser('remove', description='remove repo(s) from a group.')
```
Comment (nosarthur) [inline gita/__main__.py:374]: maybe call it something more specific, say `rmrepo`?

File: gita/__main__.py [hunk 2]
```diff
@@ -141,6 +141,14 @@ def f_group(args: argparse.Namespace):
             utils.write_to_groups_file(groups, 'w')
         else:
             utils.write_to_groups_file({gname: sorted(args.to_group)}, 'a+')
+    elif cmd == 'remove':
+        gname = args.gname
+        if gname in groups:
+            gname_repos = set(groups[gname])
+            for repo in args.from_group:
+                gname_repos.discard(repo)
+            groups[gname] = sorted(gname_repos)
```
Comment (nosarthur) [inline gita/__main__.py:150]: If `groups[gname]` is a list, we can delete the `repo` without sorting again

**Review by nosarthur — COMMENTED** [review]
Overall it looks good, please also bump the version number in `setup.py`, thanks for the contribution!

File: gita/__main__.py (hunk 1 above)
Comment (tmack0) [reply gita/__main__.py:374]: yep, renamed.

File: gita/__main__.py (hunk 2 above)
Comment (tmack0) [reply gita/__main__.py:150]: true, this was mostly a quick copypasta. patched

**Comment by nosarthur** [comment]
lgtm

**Pull request merged**